src/renderer/components/button/button.scss

```diff
@@ -21,10 +21,16 @@
   &.primary {
     background: $buttonPrimaryBackground;
   }
+
   &.accent {
     background: $buttonAccentBackground;
   }
 
+  &.light {
+    background-color: white;
```

Comment:
Is it works good with dark/light themes in Lens?

Reply:
Will check, this was copied directly from the changes Alexei made for the 3.6.10 release. Will verify.

Reply:
It looks sort of okay. I think I'll change it to "invert".